sklearn/tests/test_random_projection.py

```diff
@@ -32,11 +33,14 @@
 all_RandomProjection = all_SparseRandomProjection + all_DenseRandomProjection
 
 
-# Make some random data with uniformly located non zero entries with
-# Gaussian distributed values
-def make_sparse_random_data(n_samples, n_features, n_nonzeros, random_state=0):
+@pytest.mark.parametrize("coo_container", COO_CONTAINERS)
```

Comment:
This is not a test function: it does not start with `test_`, to `pytest.mark.*` annotations do nothing on this kind of helper functions.

You have to instead parametrize the test functions that call this `make_sparse_random_data` and pass the `coo_container` as argument to this function.

Reply:
I see. Thanks for hinting me. 

I believe, that I did a mistake in `test_SparseRandomProj_output_representation` before that I have now fixed. This test makes data using the datatypes from the `coo_container` and then converts the data into the datatypes from the `csr_container` to also do some checks.